package.json

```diff
@@ -87,7 +87,7 @@
 		"clean:packages": "yarn workspaces foreach --all --parallel --verbose --exclude 'wp-calypso' run clean",
 		"clean:public": "rm -rf public",
 		"clean:translations": "rm -rf build/strings calypso-strings.pot chunks-map.*.json || true",
-		"components:storybook:start": "echo 'Deprecated, run `yarn workspace @automattic/components run storybook` instead'",
+		"components:storybook:start": "yarn workspace @automattic/components run storybook",
 		"composite-checkout:storybook:start": "echo 'Deprecated, run `yarn workspace @automattic/composite-checkout run storybook` instead'",
```

Comment:
By the way, the "Deprecated" messages for similar sub-Storybook commands have been around for two years. Time to remove? Or if not, we might as well alias them to the command to "run instead" 😄 (cc @manzoorwanijk)

Reply:
We can probably remove it?